modelon/impact/client/client.py

```diff
@@ -725,3 +725,50 @@ def get_published_workspaces_client(self) -> PublishedWorkspacesClient:
 
         """
         return PublishedWorkspacesClient(self._sal)
+
+    def get_me(self) -> User:
+        """Return the User class object for the logged in user.
+
+        Example::
+
+            user = client.get_me()
+            user_tenant_group_name = user.tenant.group_name
+
+        """
+        user = self._sal.users.get_me()["data"]
+        return User.from_dict(user)
+
+
+@dataclass
+class Tenant:
+    id: str
+    group_name: str
+
+    @classmethod
+    def from_dict(cls, tenant_data: Dict[str, Any]) -> Tenant:
+        return Tenant(id=tenant_data['id'], group_name=tenant_data['groupName'])
+
+
+@dataclass
+class User:
```

Comment:
Any potential benefit if we used a pydantic model instead?

Reply:
Probably only for a utility funtion for parsing the dict, we would still need to define the structure(autogenerting from openapi is an option but would contain camelCase attributes). Also we would need to add a pydantic dependency just for this class if we wish to use pydantic.